app/components/Views/AssetDetails/AssetDetailsActions/AssetDetailsActions.styles.ts

```diff
@@ -0,0 +1,36 @@
+import { Theme } from '@metamask/design-tokens';
+import { StyleSheet } from 'react-native';
+
+const styleSheet = (params: { theme: Theme }) => {
+  const { theme } = params;
+  return StyleSheet.create({
+    activitiesButton: {
+      display: 'flex',
+      flexDirection: 'row',
+      justifyContent: 'center',
```

Comment:
```suggestion
      justifyContent: 'space-between',
```
I'm confirming with @amandaye0h but I think we may want to change this as well

Reply:
`space-evenly` looks the best!